pkg/replication/replication_mode.go

```diff
@@ -477,7 +488,7 @@ func (m *ModeManager) tickUpdateState() {
 			m.drSwitchToAsync(storeIDs[primaryUp])
 		}
 	case drStateAsync:
-		if canSync {
+		if canSync && m.drDurationSinceAsyncStart() > m.config.DRAutoSync.WaitRecoverTimeout.Duration {
```

Comment:
Is this config only used to switch from `async` to `syncRecover`?

Reply:
yes!